L79-cpp-byte-buffer-slice-methods.md

```diff
@@ -0,0 +1,53 @@
+C++ API changes on ByteBuffer and Slice
+----
+* Author(s): 
+* Approver: vjpai
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2021-04-19
+* Discussion at: https://groups.google.com/g/grpc-io/c/OsAYW1mDJ9w
+
+## Abstract
+
+For the better integration between gRPC and FlatBuffers, additional methods will be added to `grpc::ByteBuffer` and `grpc::Slice` allowing FlatBuffers to access data without copying data.
```

Comment:
Maybe make it more general and say  "To better support custom serialization protocols besides protobuf ... allowing those serialization protocols to access...."

Reply:
Done!